### PR TITLE
Added Traditional Chinese (zh_tw.json)

### PR DESCRIPTION
**Pull request by dirtTW**
Added Traditional Chinese translations

### CODE REVIEW

File: src/main/resources/assets/advancedhud/lang/zh_tw.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "text.autoconfig.advancedhud.title": "Advanced-HUD",
+  "text.autoconfig.advancedhud.option.enabled": "啟用",
+  "text.autoconfig.advancedhud.option.drawWithShadows": "文字陰影",
+  "text.autoconfig.advancedhud.option.textSpacing": "文字間距",
+  "text.autoconfig.advancedhud.option.textColor": "文字顏色",
+  "text.autoconfig.advancedhud.option.textAlpha": "文字透明度",
+  "text.autoconfig.advancedhud.option.offsetTop": "頂部偏移量",
+  "text.autoconfig.advancedhud.option.offsetLeft": "左側偏移量",
+  "text.autoconfig.advancedhud.option.holdKeyToShowFps": "「按鍵顯示 HUD」 模式",
```
Comment (notlin4) [inline src/main/resources/assets/advancedhud/lang/zh_tw.json]:
```suggestion
  "text.autoconfig.advancedhud.option.holdKeyToShowFps": "「按住按鍵顯示 HUD」模式",
```

File: src/main/resources/assets/advancedhud/lang/zh_tw.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "key.advancedhud.category": "Advanced-HUD",
+  "key.advancedhud.toggleFpsOverlay": "切換 HUD 疊加層",
```
Comment (notlin4) [inline src/main/resources/assets/advancedhud/lang/zh_tw.json]:
```suggestion
  "key.advancedhud.toggleFpsOverlay": "切換 HUD 覆蓋層",
```

File: src/main/resources/assets/advancedhud/lang/zh_tw.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  "text.autoconfig.advancedhud.option.enableFpsHud": "顯示 FPS",
+  "text.autoconfig.advancedhud.option.fps": "FPS",
+  "text.autoconfig.advancedhud.option.enableCooperativesHud": "顯示 座標",
```
Comment (notlin4) [inline src/main/resources/assets/advancedhud/lang/zh_tw.json]:
```suggestion
  "text.autoconfig.advancedhud.option.enableCooperativesHud": "顯示座標",
```

File: src/main/resources/assets/advancedhud/lang/zh_tw.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+  "text.autoconfig.advancedhud.title": "Advanced-HUD",
+  "text.autoconfig.advancedhud.option.enabled": "啟用",
```
Comment (notlin4) [inline src/main/resources/assets/advancedhud/lang/zh_tw.json]:
```suggestion
  "text.autoconfig.advancedhud.option.enabled": "已啟用",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Comment by Razuuu** [comment]
What's about [this](https://github.com/Razuuu/advanced-hud-fabric/pull/3/files/8d1e409b328e94816e3d13303a62f09b1451f1f9)?

**Comment by dirtTW** [comment]
> What's about [this](https://github.com/Razuuu/advanced-hud-fabric/pull/3/files/8d1e409b328e94816e3d13303a62f09b1451f1f9)?

i think it's ready to merged

**Review by notlin4 — APPROVED** [review]

**Comment by Razuuu** [comment]
Cherry picked it, commit is [5389a33](https://github.com/Razuuu/advanced-hud-fabric/commit/5389a33eab525a8d77461e82cd1b33e881dfeff1).

**Pull request closed without merging**